### ISSUE TITLE
 a nice work..   how long have  you put your heart on this job?

### ISSUE THREAD

**Issue by henbucuoshanghai** [xingyizhou/CenterTrack#1]

**Comment by xjsxujingsong** [comment]
After I red CenterNet a few days ago, I suddenly realized it could be extended to MOT. Glad to see the original author does this work. Brilliant work and I think it will refersh MOT and lead another direction on it.

**Comment by taofuyu** [comment]
> After I red CenterNet a few days ago, I suddenly realized it could be extended to MOT. Glad to see the original author does this work. Brilliant work and I think it will refersh MOT and lead another direction on it.

To reconstruct 3Dbox, do you use camera intrinsic?

**Comment by xingyizhou** [comment]
@henbucuoshanghai we had this idea in mind right after we released CenterNet, but it took some time for us to start the project and getting familiar with the tracking community. It didn't take long to see the prior heatmap/ tracking offset ideas work. I would say one tricky thing in this project (as well as the tracking community) is how to set up the validation sets for ablation, as all the popular datasets do not provide an official one. See Section 5.1 for details. It took us long to figure this out. 

@xjsxujingsong Thank you and sorry to "scoop" your idea.

@taofuyu Yes we did use the camera intrinsic in testing. We did not use it explicitly in training, where we directly learn the absolute depth.
